content/en/docs/Kyverno CLI/_index.md

```diff
@@ -224,3 +224,159 @@ policies:
 ```
 kyverno apply /path/to/add_network_policy.yaml --resource /path/to/required_default_network_policy.yaml -f /path/to/value.yaml
 ```
+
+#### Policy Report
+
+Use '--policy_report' with apply command to generate policy report.
+
+Input combination:
+
+| S.No | policy        | resource         | cluster   | namespace      | interpretation                                                                           |
+| ---- |:-------------:| :---------------:| :--------:| :-------------:| :----------------------------------------------------------------------------------------| 
+| 1.   | policy.yaml   | -r resource.yaml | false     |                | apply policy from file to the resource from file                                         |
+| 2.   | policy.yaml   | -r resourceName  | true      |                | apply policy from file to the resource in cluster                                        |
```

Comment:
"Apply policy from `policy.yaml` to all resources of a given kind in the cluster.